tools/vsimporter/xib2nib/src/XIBObject.h

```diff
@@ -123,9 +123,19 @@ class XIBObject {
     const char* GetString(char* pPropName, char* defaultValue);
     int GetInt(char* pPropName, int defaultValue);
     bool GetBool(char* pPropName, bool defaultValue);
+
+    template <typename TData>
+    void AddData(NIBWriter* writer, char* pPropName, const TData& data) {
+        char* dataOut = (char*)malloc(sizeof(TData) + 1);
+        dataOut[0] = 6;
```

Comment:
nvm, this just refactor the existing code. i wish old code have some comments there.

---
In reply to: [107232314](https://github.com/Microsoft/WinObjC/pull/2269#discussion_r107232314) [](ancestors = 107232314)